tests/Integration/Behaviour/Features/Context/Domain/Product/UpdatePositionFeatureContext.php

```diff
@@ -74,10 +74,12 @@ public function updateProductPosition(string $categoryReference, TableNode $tabl
     public function assertPositionInformation(string $categoryReference, TableNode $tableNode): void
     {
         $products = $this->localizeByColumns($tableNode);
-        $productRepository = CommonFeatureContext::getContainer()->get(ProductRepository::class);
+        $productRepository = CommonFeatureContext::getContainer()->get(ProductMultiShopRepository::class);
         $categoryId = new CategoryId($this->getSharedStorage()->get($categoryReference));
         foreach ($products as $product) {
             $productId = new ProductId($this->getSharedStorage()->get($product['product_reference']));
+            // @todo: getPositionInCategory seems to only be used for tests, the method shouldn't exist if it is ONLY for tests,
+            //        maybe there is other way to assert position?
```

Comment:
:up: just a note. Don't want to fix it in current PR, but it seems we have a method in production code only for test purposes, which I think isn't right

Reply:
I agree with you, perhaps open another PR for this 👍